policyengine_us/data/parameters/gov/aca/ca/gra.yaml

```diff
@@ -0,0 +1,165 @@
+description: ACA geographic rating areas for California.
+metadata:
+  reference:
```

Comment:
Could you add a breakdown metadata field? Here and elsewhere

Reply:
What is a "breakdown metadata field" and what kind of parameters should have one?

Reply:
@MaxGhenis, you haven't provided any information about how to specify a "breakdown metadata field" and I can't find any written developer information about it on the PolicyEngine-US GitHub site, so I was forced to experiment.  

The first thing I did was to consult Nikhil's `parameters/gov/hhs/.../medicaid_rating_area.yaml` file`.  However, that parameter file has **no breakdown metadata** as can be seen from this screenshot:

<img width="1475" alt="Screenshot 2023-11-14 at 9 31 00 AM" src="https://github.com/PolicyEngine/policyengine-us/assets/12170745/b1cc7367-2e09-4067-804e-fd1fedf0ed7a">


When I add breakdown metadata to the new `parameters/gov/aca/ca/gra.yaml` file, I get an error.  Here is what I added:
```
  breakdown:
    - county_fips
```
And here is the error I get:

<img width="1073" alt="Screenshot 2023-11-14 at 9 38 11 AM" src="https://github.com/PolicyEngine/policyengine-us/assets/12170745/48b2e5f8-f653-4a0a-a6dd-f5fe1c59c4ac">


The first thing to note is that the error is in the `policyengine-core` code.   
The second thing I noticed is that it appears as if the `policyengine-core` homogenize_parameters logic thinks `county_fips` is an enumeration (rather than an integer) variable.  Why would it think that?  Or maybe my interpretation is incorrect.

In conclusion, I cannot get your suggestion to add a "breakdown metadata field" to the `gra.yaml` parameter file to work.   And I'm puzzled about why you would make such a suggestion given that Nikhil's rating area parameter file does not contain any breakdown metadata.  (And there plenty of other parameter files in the code that appear to have missing breakdown metadata.).  I have no idea how to proceed.